de.tu_bs.cs.isf.e4cf.parts.project_explorer/src/de/tu_bs/cs/isf/e4cf/parts/project_explorer/wizards/drop_files/ImportDirectoryPage.java

```diff
@@ -1,32 +1,28 @@
-package de.tu_bs.cs.isf.e4cf.parts.project_explorer.wizards;
+package de.tu_bs.cs.isf.e4cf.parts.project_explorer.wizards.drop_files;
 
 import java.nio.file.Path;
 
 import org.eclipse.e4.core.contexts.IEclipseContext;
-import org.eclipse.jface.resource.ImageDescriptor;
-import org.eclipse.jface.wizard.WizardPage;
-import org.eclipse.swt.SWT;
-import org.eclipse.swt.widgets.Composite;
 
 import de.tu_bs.cs.isf.e4cf.core.gui.java_fx.util.FXMLLoader;
 import de.tu_bs.cs.isf.e4cf.parts.project_explorer.controller.CopyDirectoryController;
 import de.tu_bs.cs.isf.e4cf.parts.project_explorer.stringtable.FileTable;
 import de.tu_bs.cs.isf.e4cf.parts.project_explorer.stringtable.StringTable;
-import javafx.embed.swt.FXCanvas;
-import javafx.scene.Scene;
+import javafx.scene.Parent;
 
 /**
  * Represents a Wizard Page which lets the user choose to copy the content of a
  * directory or only the directory itself (empty).
  */
-public class ImportDirectoryPage extends WizardPage {
+public class ImportDirectoryPage {
 
 	private IEclipseContext context;
 
 	// the strategy how the dropped directories should be copied (3 states)
 	private CopyStrategy copyStrategy = CopyStrategy.EMPTY;
 
 	private FXMLLoader<CopyDirectoryController> loader;
+	// by passing the Pane of the dialog into createControl the dialog buttons become accessible to the page

```

Comment:
what is this comment referencing?

Reply:
it was a note left from older iterations, should this become necessary again. I removed  it